optimum/exporters/tasks.py

```diff
@@ -153,6 +158,13 @@ class TasksManager:
     _TRANSFORMERS_TASKS_TO_TF_MODEL_LOADERS = {}
     _LIBRARY_TO_TF_TASKS_TO_MODEL_LOADER_MAP = {}
 
+    # Torch model mappings
+    _TRANSFORMERS_TASKS_TO_MODEL_MAPPINGS = {}
+    _DIFFUSERS_TASKS_TO_PIPELINE_MAPPINGS = {}
+
+    # TF model mappings
+    _TRANSFORMERS_TASKS_TO_TF_MODEL_MAPPINGS = {}
```

Comment:
uses the mappings from diffusers and transformers to construct a `task->model_type->model_class`